test/src/main/java/org/axonframework/test/saga/SagaTestFixture.java

```diff
@@ -121,7 +120,8 @@ public SagaTestFixture(Class<T> sagaType) {
         registeredParameterResolverFactories.add(ClasspathParameterResolverFactory.forClass(sagaType));
         registeredHandlerDefinitions.add(ClasspathHandlerDefinition.forClass(sagaType));
         registeredHandlerEnhancerDefinitions.add(ClasspathHandlerEnhancerDefinition.forClass(sagaType));
-        listenerInvocationErrorHandler = new LoggingErrorHandler();
+        RecordingLoggingErrorHandler recordingLoggingErrorHandler = new RecordingLoggingErrorHandler();
+        listenerInvocationErrorHandler = recordingLoggingErrorHandler;
```

Comment:
These changes require adjustment of the JavaDoc on the `FixtureConfiguration#registerListenerInvocationErrorHandler`, since the default is changed.
Furthermore, I think it's good to be specific that changing the error handler means support for the `expectSuccessfulHandlerExecution` is lost.